command/push.go

```diff
@@ -119,23 +133,3 @@ func pushMessage(parsedURL *url.URL, token string, msg models.MessageExternal, q
 		utils.Exit1With(err)
 	}
 }
-
-func readMessage(ctx *cli.Context) string {
-	msgArgs := strings.Join(ctx.Args(), " ")
-
-	msgStdin := utils.ReadFrom(os.Stdin)
-
-	if msgArgs == "" && msgStdin == "" {
-		utils.Exit1With("a message must be set, either as argument or via stdin")
```

Comment:
This is indended. The user might want to input a multiline string to push and press ^D to end. What do you think?. `go run . push </dev/null` will error out because there is actually nothing to read from